data.access/src/test/java/CsvTests.java

```diff
@@ -0,0 +1,111 @@
+import static org.testng.Assert.assertEquals;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import org.testng.annotations.Test;
+
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvException;
+
+import data.access.Bass;
+
+public class CsvTests {
```

Comment:
Reading of csv files and hydrating objects with csv data should be performed outside of the testclass.